bloom-housing/bloom
# fix: add bengali and arabic localization

This PR addresses #4427 


## Description

In order to support Detroit we need to have Bengali and Arabic translations in core. We can't just pull over all of the translations from Detroit as some of them are Detroit specific. Also they are missing many translations for things we support in core.

We need to do the following three steps and this PR covers steps 1
1. Pull over all Bengali and Arabic translations that are used on the public site in core
2. In the new Detroit repo add the Detroit specific translations to the locale_override folder - https://github.com/bloom-housing/bloom-detroit/pull/3
3. Get translations for all missing Bengali and Arabix translations

## How Can This Be Tested/Reviewed?

In order to see Bengali translations on the site you will need to run the migrations and add "bn,ar" to the languages environment variable in site/public

You will see a decent amount of English still on the site. These have been compiled and will be covered in part 3 from above

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
